paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetResultCallback.kt

```
@@ -10,5 +10,5 @@ import com.stripe.android.ExperimentalCustomerSheetApi
 @ExperimentalCustomerSheetApi
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun interface CustomerSheetResultCallback {
-    fun onResult(result: CustomerSheetResult)
+    fun onResult(result: CustomerSheetResult?)
```

Not the nullable result again!

When would this be null?

---

Null when parcelizing :D

---

Can we return an error result instead of null? Or even cancelled?